tests/31sync/09archived.pl

```diff
@@ -117,6 +117,59 @@
       });
    };
 
+test "We should see our own leave event when rejecting an invite, ".
+    "even if history_visibility is restricted ",
+   requires => [ local_user_fixture( with_events => 0 ),
+                 local_user_fixture( with_events => 0 ),
+                 qw( can_sync ) ],
+
+   check => sub {
+      my ( $user1, $user2 ) = @_;
```

Comment:
could we call them `$invitee`, `$inviter` or something? I lost track about 15 times when reading through